doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,742 @@
+    "\n",
+    "- Use **`cmap`** for **continuous numerical values** (e.g., depth, magnitude). It maps a range of values to a color gradient.\n",
+    "- Use **`color_key`** for **categorical values** (e.g., mag_class, depth_class). It maps each category explicitly to a color.\n",
+    "\n",
+    "For example:\n",
+    "- `cmap='viridis'` is great for smooth gradients like 'magnitude' or 'depth'.\n",
+    "- `color_key={'Shallow': 'blue', 'Deep': 'green'}` is best for labeled classes like 'depth_class'.\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b27eb1cc-3a6e-400b-816d-094a22bffac9",
+   "metadata": {},
+   "source": [
+    "(option-clim)=\n",
+    "## `clim`\n",
+    "\n",
+    "The `clim` option sets the lower and upper bounds of the color scale for continuous color mapping. It accepts a tuple like `(min, max)` and is useful when:\n",
+    "\n",
+    "- You want consistent color scaling across multiple plots.\n",
+    "- You want to clip outliers or focus on a specific data range.\n",
+    "\n",
+    "This option is most effective when used with **gridded plots** like `image`, `heatmap`, or rasterized plots (`datashade=True`, `rasterize=True`). It works especially well with **xarray datasets** or plots where a **colorbar** is enabled.\n",
+    "\n",
+    "In standard scatter plots or overlays created from tabular (pandas) data, `clim` may have no visible effect unless color mapping and a colorbar are explicitly involved.\n",
+    "\n",
+    "If `clim` is not specified, the color scale is inferred from the data — or from the 2nd and 98th percentiles when `robust=True`.\n",
+    "\n",
+    "::: {seealso}\n",
+    "[`robust`](option-robust).\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74d74cec-a37a-4be3-b8e2-9a607e20590d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.xarray  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.air_temperature(\"xarray\").sel(time=\"2014-02-25 12:00\")\n",
+    "\n",
+    "plot1 = df.hvplot.image(clim=(250, 280), title=\"Colorbar clipped at clim values\", frame_width=300)\n",
+    "plot2 = df.hvplot.image(title=\"Default colorbar scale\", frame_width=300)\n",
+    "\n",
+    "plot1 + plot2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35b5e132-8fde-4637-8289-e29bc0bb4cb5",
+   "metadata": {},
+   "source": [
+    "(option-cnorm)=\n",
+    "## `cnorm`\n",
+    "\n",
+    "The `cnorm` option controls how data values are mapped to colors in a colormap. It affects the distribution of colors across the range of values.\n",
+    "\n",
+    "Accepted values:\n",
+    "- 'linear' (default): evenly maps values across the colormap.\n",
+    "- 'log': applies logarithmic scaling, useful for data with large dynamic range.\n",
+    "- 'eq_hist': uses [histogram equalization](https://en.wikipedia.org/wiki/Histogram_equalization) to emphasize contrast in sparse or skewed data."
```

Comment:
Is there a place to point to in HoloViews or Datashader docs? This wikipedia page doesn't help much.

Reply:
OK. There's a link to datashader docs I can use.